docs/docsite/rst/shared_snippets/installing_multiple_collections.txt

```diff
@@ -32,7 +32,16 @@ file used in older Ansible releases.
         version: 0.9.3
         source: https://galaxy.ansible.com
 
+To install both roles and collections at the same time with one command, run the following:
+
+.. code-block:: bash
+
+    $ ansible-galaxy install -r requirements.yml
+
+Running ``ansible-galaxy collection install -r`` or ``ansible-galaxy role install -r`` will only install collections,
+or roles respectively.
+
 .. note::
-    While both roles and collections can be specified in one requirements file, they need to be installed separately.
-    The ``ansible-galaxy role install -r requirements.yml`` will only install roles and
-    ``ansible-galaxy collection install -r requirements.yml -p ./`` will only install collections.
+    Installing both roles and collections from the same requirements file will not work when specifying a custom path
+    to install either collections or roles to. In this scenario the collections will be skipped and the command will
+    process each like like ``ansible-galaxy role install`` would.
```

Comment:
```suggestion
    process each like ``ansible-galaxy role install`` would.
```

Little nit about the sentence before - does 'Installing both roles and collections from the same requirements file will not work when specifying a custom installation path.' read better?